msgpack5/index.d.ts

```diff
@@ -0,0 +1,51 @@
+// Type definitions for msgpack5 v3.4.0
+// Project: https://github.com/mcollina/msgpack5/
+// Definitions by: Wonshik Kim <https://github.com/wokim/>, Kei Son <https://github.com/heycalmdown/>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+import stream = require('stream');
+import bl = require('bl');
+  
+interface Options {
```

Comment:
Use two separate interfaces `MsgPackOptions` and `EncodeDecodeOptions`. Or just write this definition inline as `declare function msgpack5(opts? { forceFloat64: boolean; compatibilityMode: boolean }): msgpack5.MessagePack;`.
Also, usually options are optional properties -- is `msgpack5({ forceFloat64: true })` without `compatibilityMode` specified allowed?